### PR TITLE
[IMP] website: introduce s_striped snippet

### PR DESCRIPTION
**Pull request by chgo-odoo**
This commit introduces a new snippet named `s_striped`.

task-4108307
Part of task-4077427

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/s_striped.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<odoo>
+
+<template id="s_striped" name="Striped section">
+    <section class="s_striped o_cc o_cc5 pt56 pb96" data-oe-shape-data="{'shape':'web_editor/Origins/04_001','colors':{'c3': '#FFFFFF'},'flip':['y'], 'showOnMobile':true}">
+        <div class="o_we_shape o_web_editor_Origins_04_001 o_shape_show_mobile" style="background-image: url(&quot;/web_editor/shape/web_editor%2FOrigins%2F04_001.svg?c3=%23FFFFFF&amp;flip=y&quot;); background-position: 50% 100%;"/>
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 pt24 pb24">
+                    <h2 style="text-align: center;">The evolution of our company</h2>
+                    <p class="lead" style="text-align: center;">Learn about the key decisions that have shaped our identity.    </p>
```
Comment (bvr-odoo) [inline addons/website/views/snippets/s_striped.xml:11]: they are empty spaces in this line

File: addons/website/views/snippets/s_striped.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_striped" name="Striped section">
+    <section class="s_striped o_cc o_cc5 pt56 pb96" data-oe-shape-data="{'shape':'web_editor/Origins/04_001','colors':{'c3': '#FFFFFF'},'flip':['y'], 'showOnMobile':true}">
```
Comment (bvr-odoo) [inline addons/website/views/snippets/s_striped.xml:5]: Missing `style="position: relative;"`
It is added to sections containing a shape by the editor ;)

**Review by bvr-odoo — CHANGES_REQUESTED** [review]
@chgo-odoo A few changes to make here, thank you.

File: addons/website/views/snippets/snippets.xml [hunk 3]
```diff
@@ -66,6 +66,9 @@
                 <t t-snippet="website.s_cta_box" string="Box Call to Action" group="content">
                     <keywords>CTA, button, btn, action, engagement, link, offer, appeal, call to action, prompt, interact, trigger</keywords>
                 </t>
+                <t t-snippet="website.s_striped" string="Striped section" group="content">
+                    <keywords>CTA, button, btn, action, engagement, link, offer, appeal, call to action, prompt, interact, trigger</keywords>
```
[the remark this answers is not in the document]
Comment (chgo-odoo) [reply addons/website/views/snippets/snippets.xml:70]: Indeed.. 🤦

File: addons/website/views/snippets/s_striped.xml (hunk 2 above)
Comment (chgo-odoo) [reply addons/website/views/snippets/s_striped.xml:5]: Thanks, didn't pay attention to that, might need to apply this as well to some of my others PR's 😅

**Comment by chgo-odoo** [comment]
Thanks @bvr-odoo, changes have been applied and I also rebased the branch to solve the conflicts. Could you have a look please?

Thanks 👍

**Comment by bvr-odoo** [comment]
@chgo-odoo I ve just seen that another new snippet is almost the same than this one => `Striped Center Top`
Is it wanted ?

**Comment by chgo-odoo** [comment]
Hello, it's very similar to `s_striped_centertop` indeed. TL;DR, this snippet was the first one created, but `s_striped_centertop` was created afterward, making this one look a bit like a clone. 

This snippet was created after a discussion on task-4105455, which resulted in moving this snippet (let's call it the clone) to the content category, as it makes sense there.

**Comment by bvr-odoo** [comment]
> Hello, it's very similar to `s_striped_centertop` indeed. TL;DR, this snippet was the first one created, but `s_striped_centertop` was created afterward, making this one look a bit like a clone.
> 
> This snippet was created after a discussion on task-4105455, which resulted in moving this snippet (let's call it the clone) to the content category, as it makes sense there.

ok :+1: , I would have at least changed the image to differentiate them more easily... but it's up to you to decide :wink:

**Review by bvr-odoo — APPROVED** [review]
LGTM :+1: , thanks for your changes

**Comment by chgo-odoo** [comment]
@bvr-odoo sorry for the delay, I was finishing another snippet at the same time 😅 It's indeed better if we change the image on this one, just so the user doesn't feel like there are duplicated snippets.

I used the `s_text_cover image` which is currently not adapted within themes (not related to the snippet redesign) but I have a PR that aims to fix different issues on the `s_text_cover` snippet which will add images within themes so we'll be covered 🙂

**Comment by bvr-odoo** [comment]
@chgo-odoo Thanks, the PR is still in draft ;)

**Comment by bvr-odoo** [comment]
@robodoo r+

**Comment by sobo-odoo** [comment]
@robodoo delegate=bvr-odoo

**Comment by bvr-odoo** [comment]
@robodoo r+

**Pull request closed without merging**